components/resources/ai_chat_agent_ntp_strings.grdp

```diff
@@ -1,6 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <grit-part>
-  <message name="IDS_AI_AGENT_NTP_TITLE" desc="Title for the Brave AI Agent mode New Tab Page" formatter_data="webui=AiChatAgentNtp">
+  <message name="IDS_AI_AGENT_NTP_AGENT_MODE_BUTTON_LABEL" translateable="true" desc="Agent mode button text for the Brave AI Agent mode New Tab Page" formatter_data="webui=AiChatAgentNtp">
     Agent mode
   </message>
+  <message name="IDS_AI_AGENT_NTP_ABOUT_THE_LEO_PROFILE_CARD_TITLE" translateable="true" desc="About the Leo profile text for the Brave AI Agent mode New Tab Page" formatter_data="webui=AiChatAgentNtp">
+    About the Leo profile
+  </message>
```

Comment:
desc should mention it's a title, otherwise context for translation might be misinterpreted